internal/controllers/machinedeployment/machinedeployment_rolling_test.go

```diff
@@ -173,6 +173,50 @@ func TestReconcileNewMachineSet(t *testing.T) {
 			},
 			error: nil,
 		},
+		{
+			name: "RollingUpdate strategy: while machine in old machine set is being deleted, machine in new machineset is not scaled in",
```

Comment:
did you mean "machine in new machineset is not scaled **out**"?

To be consistent with other test casas, how about: "RollingUpdate strategy: Scale up accounts for deleting Machines to honour maxSurge"?

Reply:
I think 'scale out' is correct when talking about an individual machine, so I did mean 'scale out' here. But the terminology in all comments refers to 'scale up / scale down' a MachineSet, so let's match that. 

Your test case name is more succinct and clearer, thanks! I've pushed that suggestion.